SPECS/cockpit/cockpit.spec

```diff
@@ -0,0 +1,90 @@
+%define _unpackaged_files_terminate_build 0
```

Comment:
>_unpackaged_files_terminate_build [](http://example.com/codeflow?start=8&length=33)

Please remove unnecessary files in the `%install` section instead. Ideally with a short comment explaining why we don't need them. Doing thing this way makes it easier to miss files we might actually need in the future updates.

Reply:
I rebuilt the cockpit spec yesterday with this flag set to 1, and it built successfully. So I guess I already had all the files listed in %files and can safely remove this line.